frontend/src/components/Meal/Day.js

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from "react";
+import axios from "axios";
+import { DateTime } from "luxon";
+import { APP_URL } from "../../config";
+import Box from "@material-ui/core/Box";
+import Collapse from "@material-ui/core/Collapse";
+import IconButton from "@material-ui/core/IconButton";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
+import KeyboardArrowDownIcon from "@material-ui/icons/KeyboardArrowDown";
+import KeyboardArrowUpIcon from "@material-ui/icons/KeyboardArrowUp";
+import { getBackendAuthToken } from "../../session/localStorage";
+import { useDayStyles } from "../../styles/theme";
+
+function getCardSubtitleAsCalories(meals) {
+  var total = 0;
+  var i;
+  for (i = 0; i < meals.length; i++) {
+    total = total + meals[i].total_kcal;
+  }
+  return total + " kcal";
+}
+
+function getMealNameAsTime(mealDatetimeStr) {
+  var hour = Number(mealDatetimeStr.substr(11, 2));
```

Comment:
Similary as above, please reuse using const.
Those data operations can also be done using Luxon library you use, please rewrite using them. https://moment.github.io/luxon/docs/class/src/datetime.js~DateTime.html#instance-get-hour